README.md

```diff
@@ -28,6 +28,7 @@ After you're finished please remove all the comments and instructions!
 
   - [🛠 Built With ](#-built-with-)
     - [Tech Stack ](#tech-stack-)
+    - [Key Features](#key-features)
   - [💻 Getting Started ](#-getting-started-)
     - [Prerequisites](#prerequisites)
     - [Setup](#setup)
```

Comment:
- [ ] The License link in the README file is broken, Please make sure to fix this. It will make the README file more professional.